frontend/src/components/GoalsTable/GoalRow.css

```diff
@@ -0,0 +1,61 @@
+.tta-smarthub--goal-row td:first-child {
+  position: relative;
+}
+
+.tta-smarthub--goal-row  .smart-hub--context-menu {
+  width: 180px;
+  /*box-shadow: 0.25rem 0.5rem 0 rgb(0 0 0 / 10%);*/
+  box-shadow: 0 0 0 1px hsla(0, 0%, 0%, 0.1),0 4px 11px hsla(0, 0%, 0%, 0.1);
+}
+
+/**
```

Comment:
I don't think we need any of the export-reports css here

Reply:
All export CSS has been removed.